lte/gateway/c/oai/tasks/nas/emm/msg/SecurityModeCommand.c

```diff
@@ -121,6 +122,22 @@ int decode_security_mode_command(
             SECURITY_MODE_COMMAND_NONCEMME_PRESENT;
         break;
 
+      case SECURITY_MODE_COMMAND_REPLAYED_UE_ADDITIONAL_SECURITY_CAPABILITY_IEI:
+        if ((decoded_result = decode_ue_additional_security_capability(
+                 &security_mode_command
+                      ->replayedueadditionalsecuritycapabilities,
+                 SECURITY_MODE_COMMAND_REPLAYED_UE_ADDITIONAL_SECURITY_CAPABILITY_IEI,
+                 buffer + decoded, len - decoded)) <= 0)
+          return decoded_result;
```

Comment:
Please add braces around If loops, even if they are one-line statements

```suggestion
                 buffer + decoded, len - decoded)) <= 0) {
          return decoded_result;
          }
```

Reply:
Added braces.